### PR TITLE
Add libgcc and remove LICENSE words

### PR DESCRIPTION
**Pull request by lpantano**
I add a diff between the original and the final meta. yaml to check the license section, just incase the code remove something it shouldn't.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -1,24 +1,33 @@
...
     sed -i '/^\s*#.*$/ d' $fn/meta.yaml
     sed -i '/^mv\s.*$/ d' $fn/build.sh
     sed -i '/^grep\s.*$/ d' $fn/build.sh
     sed -i '/^#\s.*$/ d' $fn/build.sh
     sed -i '/^@.*$/ d' $fn/bld.bat
     sed -i '/^$/{N;/^\n$/d;}' $fn/meta.yaml
     sed -i '/^$/{N;/^\n$/d;}' $fn/build.sh
+    sed -i 's/ + file LICENSE//' $fn/meta.yaml
+    sed -i 's/ \| file LICENSE//' $fn/meta.yaml
+    if grep -lq "\- gcc"  $fn/meta.yaml ; then sed  -i 's/run:/run:\n   - libgcc               # [not win]/' $fn/meta.yaml ; fi
```
Comment (bgruening) [inline run.sh]: behind libgcc only two spaces

File: run.sh [hunk 2]
```diff
@@ -1,24 +1,33 @@
...
     # skip win builds
     sed -i 's/number: 0/number: 0\n  skip: true  # [win32]/g' $fn/meta.yaml
 
     sed -i -e :a -e '/^\n*$/{$d;N;};/\n$/ba' $fn/bld.bat
     sed -i -e :a -e '/^\n*$/{$d;N;};/\n$/ba' $fn/meta.yaml
     sed -i -e :a -e '/^\n*$/{$d;N;};/\n$/ba' $fn/build.sh
-    cat extra.yaml >> $fn/meta.yaml
+    if grep -lvq "extra:" $fn/meta.yaml ; then  cat extra.yaml >> $fn/meta.yaml ; fi
```
Comment (bgruening) [inline run.sh]: Is there ever an extra section already?

Comment (lpantano) [inline run.sh]: I was running the script more than once, but I think we can assume it will run once.

Comment (bgruening) [inline run.sh]: Think so too. Can you remove this perl magic :)

**Comment by bgruening** [comment]
Cool, thanks!

**Pull request merged**